UI/cypress/e2e/mobile.js

```diff
@@ -0,0 +1,39 @@
+import { Given, When, Then, Step } from "@badeball/cypress-cucumber-preprocessor";
+
+Then("a user is using a {string}", (device) => {
+    cy.viewport(428, 926);
```

Comment:
Add a case or if statement for the three[ GOV.UK frontend media queries options](https://frontend.design-system.service.gov.uk/sass-api-reference/#media-queries) and use the pixel widths they use

Reply:
Added three media default widths